media-libs/webvfx/webvfx-1.1.0.ebuild

```diff
@@ -0,0 +1,49 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit flag-o-matic qmake-utils toolchain-funcs
+
+DESCRIPTION="Video effects library based on web technologies"
+HOMEPAGE="https://github.com/mltframework/webvfx/"
+SRC_URI="https://github.com/mltframework/${PN}/releases/download/1.1.0/${P}.txz"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE=""
+
+RDEPEND="
+	dev-qt/qtcore:5
+	dev-qt/qtdeclarative:5
+	dev-qt/qtgui:5
+	dev-qt/qtnetwork:5
+	dev-qt/qtopengl:5
+	dev-qt/qtquickcontrols:5[widgets]
+	dev-qt/qtwebkit:5
+	dev-qt/qtwidgets:5
+	media-libs/mlt
+"
+DEPEND="${RDEPEND}"
+
+src_prepare() {
+	default
+
+	find -name "*.pro" | xargs sed -i \
+		-e "s/\(system.*\)pkg-config/\1$(tc-getPKG_CONFIG)/"
```

Comment:
`|| die`

Reply:
Is it really necessary? `find ... -exec` does not return bad status.